_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -0,0 +1,300 @@
+
+Artsy puts a lot of priority on references,  Check out [Brennan Moore's article](https://www.zamiang.com/posts/post/2015/07/05/learning-from-artsy-how-to-hire-awesome-engineers/) on this, a lot of the behind the scenes emails afterwards uses quotes taken from references as examples of why someone should be hired. I don't think this will be unique to Artsy. References should be your cheerleaders. In an ideal world you should be looking for references like the ones we got for Maxim:
+
+> However, if I had to fire people gradually, I'd fire her last. her skill-set is so valuable, and so scarce that it's super valuable. I'd fire me before her.
+
+> She was the glue that held the team together - could talk at a different level to each contributor. Great intuition, could put in a room with anyone and they could understand how to get their bits done.
+
+I respect that no-one is in control of other people, so yes, references are a bit of a wild card from a junior's perspective. However, hopefully you've had a collection of positive interactions with people who can talk about that. It's not about your programming prowess at this point, it's about how you work with others and character reference.
+
+### On the Aftermath
+
+So, interviews are over. What is happening behind the scenes? After your interviews are done, there will be a flurry of behind the scenes emails. From my experience at Artsy, it probably takes a few days to get enough of a consensus around a yay/nay. Someone should be keeping the applicant up to date, even if it's a matter of "not yet, but we're talking."
+
+This bit is hard, because you're in limbo, and it can take a while. I think with Sarah this took about a week and a half, from her final interviews to being able to send her an offer. Which I'm sure for her weren't great, she could be so close to an offer - or weeks wasted on moving towards another.
+
+### On the Launch Pad
+
+OK, so you've been given an offer. If it's a startup, you might be offered equity. If you don't know this world, that's OK, it's hard to give advice here - but [this](https://github.com/jlevy/og-equity-compensation) seems to be best resource I know about it all. I started with no knowledge, and eventually got a reasonable understanding. If you want one sentence from me, "equity is a risk, be damn sure you think the company is going to go somewhere." I opted for a chunk of equity in Artsy, but I've worked at places where I've taken the minimum option.
+
+Ideally you are presented with a great offer, I've never negotiated salary and we don't do it at Artsy as it [introduces bias](https://en.wikipedia.org/wiki/Gender_pay_gap_in_the_United_States#Negotiating_salaries), so I can't offer much there.
+
```

Comment:
I'm not sure if it's worth including this, but I have often told people the following:

Whether someone wants to negotiate or not is up to you, but the only way to know if you want to or not is roughly knowing what the market is. It doesn't hurt to politely ask. If you know people in the industry, mentors, other people that hire, you can carefully ask or mention that you're thinking of a figure and whether they think it fits with the current landscape. You often don't have to discuss exact figures with friends, but ranges are useful so at least you know what to compare things to. Do keep in mind the company (small / large / young / old), the location, experience, other benefits (pension/health/food/etc - these are often calculated in the full package), and current economy.

The tl;dr there is (and I say this because I used to be like this :)) don't be scared to ask politely to figure out what is going on around salary. Maybe worth including? Unfortunately a lot of places do still require you to negotiate a bit :(. If you'd rather not (as the paragraph is nice and concise), that's all good.